convert_v2.py

```diff
@@ -0,0 +1,110 @@
+import sys
+import argparse
+import gzip
+import shutil
+import tarfile
+import rarfile
+import patoolib
+import unrar
+
+class FilesFamily:
```

Comment:
In python2 classes should inhert from object (unless you inhert from another class)
```python
class FilesFamily(object)
```